inventory.lua

```diff
@@ -146,6 +146,13 @@ local tube_insert_object = function(pos, _, original_stack, direction)
 	return stack
 end
 
+local mcl_bg = ""
+
+if minetest.get_modpath("mcl_formspec") then
+	mcl_bg = mcl_formspec.get_itemslot_bg(0,1,8,4)..
+		mcl_formspec.get_itemslot_bg(0,6,8,4)
+end
```

Comment:
This variable can be re-used for `default` to replace the lines 177-179.

Reply:
I don't quite understand what you mean, I think I have solved it now

Reply:
This is what I meant:
```Lua
local formspec_header = ""
if minetest.get_modpath("mcl_formspec") then
	formspec_header = mcl_formspec.get_itemslot_bg(0,1,8,4)..
		mcl_formspec.get_itemslot_bg(0,6,8,4)
end
if minetest.get_modpath("default") then
	-- specified by formspec prepends
end
```

and below:
```Lua
meta:set_string("formspec", "size[8,10]"..
			formspec_header ..
			"label[0,0;" .. S("Digiline Chest") .. "]" ..
```

The `default.gui_*` strings are deprecated and are now contained as formspec prepends.